.snapshots/should_return_an_error_when_an_ADT_defines_a_constructor_with_an_unbound_variable/golden

```diff
@@ -1 +0,0 @@
-Left (InferError (UnboundVariable "b") NoReason)
```

Comment:
This was actually wrong, it should be possible to define a type like this:

```
data User = User String a
```

Where a could be anything.